### PR TITLE
MMT-1930 Updating UMM-V

### PR DESCRIPTION
**Pull request by ryanmiller-1**

### CODE REVIEW

File: lib/assets/schemas/variables/umm-var-form.json [hunk 1]
```diff
@@ -453,9 +469,13 @@
                   "items": [
                     {
                       "type": "section",
-                      "items": [{"key": "/MeasurementSource", "field": true, "type": "UmmSelect"}]
+                      "items": [{"key": "/MeasurementObjectURI", "field": true, "htmlClass": "uri"}]
```
Comment (ryanmiller-1) [inline lib/assets/schemas/variables/umm-var-form.json]: remove htmlClass

File: lib/assets/schemas/variables/umm-var-search-results-json-schema.json [hunk 2]
```diff
@@ -0,0 +1,131 @@
+{
```
Comment (ryanmiller-1) [inline lib/assets/schemas/variables/umm-var-search-results-json-schema.json]: do we need any of this?

File: app/concerns/controlled_keywords.rb [hunk 3]
```diff
@@ -162,4 +162,22 @@ def set_country_codes
     united_states = country_codes.delete(Carmen::Country.named('United States'))
     @country_codes = country_codes.unshift(united_states)
   end
+
+  def set_measurement_names
```
Comment (ryanmiller-1) [inline app/concerns/controlled_keywords.rb]: add comment to explain structure

**Comment by ryanmiller-1** [comment]
PO feedback was to remove the auto-selection behavior from the non-required fields.  This answers the question populating multiple fields with the same value.

File: lib/assets/schemas/variables/umm-var-form.json [hunk 4]
```diff
@@ -583,5 +603,47 @@
         }
       ]
     }
-  ]
+  ],
+  "definitions": {
```
Comment (charleshuang80) [inline lib/assets/schemas/variables/umm-var-form.json]: what was the reason to create this definitions section and define the two fields here?

Comment (ryanmiller-1) [reply lib/assets/schemas/variables/umm-var-form.json]: At some point while trying to understand how to make the modifications I wanted, separating them worked better.  I have updated the schema to not include the definitions section now.

**Pull request merged**